models.py

```diff
@@ -284,7 +285,8 @@ def forward(
 	) -> shaping.BCt:
 
 		if self.frontend is not None:
-			mask = temporal_mask(x, compute_output_lengths(x, xlen)) if xlen is not None else None
+			ol = compute_output_lengths(x, xlen)
+			mask = temporal_mask(x.squeeze(1), ol) if xlen is not None else None
```

Comment:
Please eliminate duplicate `x.squeeze(1)` by putting it in a variable, e.g. `x = x.squeeze(1)` if x is not used below

Reply:
done